boltdesignsystem/bolt
# DS-537: Standarize schema and twig for bolt-banner

## Jira

https://pegadigitalit.atlassian.net/browse/DS-537
https://pegadigitalit.atlassian.net/browse/DS-538

## Summary

These two tickets were done at the same time due they are related to each other. Files: `banner.schema.js`, `banner.twig` inside `packages/components/bolt-banner/src/` and `banner.js` inside `__tests__` were reviewed.

## Details

Component was checked with master tickets.

## How to test

Check if it is up to date with standards mentioned in master tickets:

Refer to the Standardize Schema ticket: https://pegadigitalit.atlassian.net/browse/DS-369
Refer to the Standardize Twig ticket: https://pegadigitalit.atlassian.net/browse/DS-370
Refer to the Standarize Test ticket: https://pegadigitalit.atlassian.net/browse/DS-413
